### PR TITLE
Add Aten::upsample nearest 3d series ops

### PR DESCRIPTION
**Pull request by chunhuanMeng**
- [x] _upsample_nearest_exact3d
- [x] _upsample_nearest_exact3d_backward
- [x] upsample_nearest3d
- [x] upsample_nearest3d_backward

### CODE REVIEW

File: src/ATen/native/xpu/sycl/UpSampleNearest3dKernels.cpp [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+  int output_width = output_size[2];
+
+  int nbatch = input_size[0];
+  int channels = input_size[1];
+  int input_depth = input_size[2];
+  int input_height = input_size[3];
+  int input_width = input_size[4];
+
+  Tensor grad_output = grad_output_.contiguous();
+  unsigned int n = grad_input.numel() / nbatch;
```
Comment (xytintel) [inline src/ATen/native/xpu/sycl/UpSampleNearest3dKernels.cpp:372]: Need to have 32bit check for `grad_input` and `grad_output`.

**Comment by xytintel** [comment]
Please paste the test cases in the comments.

File: src/ATen/native/xpu/UpSample.h [hunk 2]
```diff
@@ -50,6 +50,48 @@ inline C10_UNUSED std::array<int64_t, 4> upsample_2d_common_check(
   return {nbatch, channels, output_height, output_width};
 }
 
+inline C10_UNUSED std::array<int64_t, 5> upsample_3d_common_check(
```
[the remark this answers is not in the document]
Comment (chunhuanMeng) [reply src/ATen/native/xpu/UpSample.h:53]: There is a compilation conflict, we implemented `upsample_2d_common_check` in this file too.

**Comment by chunhuanMeng** [comment]
image

**Pull request merged**